app/components/organization/AgentsCount.js

```diff
@@ -7,52 +9,61 @@ import { formatNumber } from '../../lib/number';
 class AgentsCount extends React.Component {
   static propTypes = {
     organization: React.PropTypes.shape({
-      id: React.PropTypes.string.isRequired,
       agents: React.PropTypes.shape({
         count: React.PropTypes.number.isRequired
       })
-    })
+    }),
+    relay: React.PropTypes.object.isRequired
   };
 
   state = {
     agentCount: this.props.organization.agents ? this.props.organization.agents.count : 0
   };
 
   componentDidMount() {
-    PusherStore.on("organization_stats:change", this.handleStoreChange);
+    PusherStore.on("organization_stats:change", this.handlePusherWebsocketEvent);
   }
 
   componentWillUnmount() {
-    PusherStore.off("organization_stats:change", this.handleStoreChange);
+    PusherStore.off("organization_stats:change", this.handlePusherWebsocketEvent);
   }
 
-  handleStoreChange = (payload) => {
-    this.setState({ agentCount: payload.agentsConnectedCount });
+  // Like in MyBuilds, we don't take the Pusher data for granted - we instead
+  // take it as a cue to update the data store backing the component.
+  handlePusherWebsocketEvent = (payload) => {
+    if (this.state.agentCount !== payload.agentsConnectedCount) {
+      this.props.relay.forceFetch();
```

Comment:
Rad - I think we may want to throttle this particular `forceFetch`. If a customer connects 1000 agents, and they all come on one by one, the browser will do 1000 odd requests.

Perhaps a throttling mechanism like:

1. `forceFetch` is called
2. Can't be called again for another 3 seconds
3. If a `forceFetch` is requested during the timeout period, one will be scheduled in when the timer runs out
4. Go to 1

Previously I would have just used the methods from `underscore`, but I don't really wanna introduce that package into frontend. Maybe there's a generic throttling lib we can use? I've wanted to throttle stuff in other places as well.